net_transport_test.go

```diff
@@ -96,7 +95,7 @@ func TestNetworkTransport_CloseStreams(t *testing.T) {
 		}
 
 		// Try to do parallel appends, should stress the conn pool
-		for i := 0; i < 5; i++ {
+		for j := 0; j < 5; j++ {
```

Comment:
this is fine here, but fwiw it can be nice to shadow `i` in the inner loop if you don't need to use `i` and `j` at the same time, it keeps the reader from looking for where they're used together.